plugins/lang-javascript/src/main/java/org/elasticsearch/script/javascript/JavaScriptScriptEngineService.java

```diff
@@ -68,7 +68,9 @@
  */
 public class JavaScriptScriptEngineService extends AbstractComponent implements ScriptEngineService {
 
-    public static final List<String> TYPES = Collections.unmodifiableList(Arrays.asList("js", "javascript"));
+    public static final String NAME = "javascript";
```

Comment:
It's still breaking because people could previously specify `"lang": "js"` and now they can't, it has to be `"lang": "javascript"`